cmd/root.go

```diff
@@ -236,13 +236,13 @@ func addCommands(groups []command.Command, executables []command.Command) {
 			Use:                v.Name(),
 			Short:              v.ShortDescription(),
 			Long:               v.LongDescription(),
-			RunE: func(cmd *cobra.Command, args []string) error {
-				err := executeCommand(group, name, args)
+			Run: func(cmd *cobra.Command, args []string) {
+				exitCode, err := executeCommand(group, name, args)
 				if err != nil && err.Error() == EXECUTABLE_NOT_DEFINED {
 					cmd.Help()
-					return nil
+					os.Exit(exitCode)
```

Comment:
I'm not too familiar with the architecture of the code, but shouldn't the actual exit done by the caller to the run method?

Reply:
You are right, this could skip the postRun process. I will make a patch on it